tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3226,7 +3226,7 @@ ruleTester.run('no-unused-prop-types', rule, {
           bar,
         };
       `,
-      parser: parsers.TYPESCRIPT_ESLINT
+      parser: parsers['@TYPESCRIPT_ESLINT']
```

Comment:
specifically, all these test cases with the old TS parser should be left untouched, and duplicated, with the duplicate using the new TS parser.

Reply:
OK! I have added it back.
But I think we might need to deprecated old TS parser someday.
The ast tree between two parser have a big difference.
It is really hard to support both parser for their own ast tree.